pyproject.toml

```diff
@@ -1,3 +1,26 @@
+[tool.poetry]
+name = "rasa-sdk"
+version = "1.6.2b1"
```

Comment:
since this now means we need to change the version in multiple places (and keep it in sync) we should add a `make release` command and a script similar to what we have in the other repos